streaming/java/streaming-api/src/main/java/org/ray/streaming/api/stream/DataStream.java

```diff
@@ -33,11 +35,19 @@ public DataStream(DataStream input, StreamOperator streamOperator) {
     super(input, streamOperator);
   }
 
+  /**
+   * Create a java stream that reference passed python stream.
+   * Changes in new stream will be reflected in referenced stream and vice versa
+   */
+  public DataStream(PythonDataStream referencedStream) {
+    super(referencedStream);
+  }
+
   /**
    * Apply a map function to this stream.
    *
    * @param mapFunction The map function.
-   * @param <R> Type of data returned by the map function.
```

Comment:
Could you update IDE settings to avoid unnecessary format changes in such a large PR for smooth review?